assets/js/admin/products-admin.js

```diff
@@ -407,10 +407,11 @@ jQuery( document ).ready( function( $ ) {
 
 			let syncValuesStatus = $syncModeSelect.map( function ( index, selectElement ) {
 
-				let $syncMode     = $( selectElement );
-				let syncModeValue = $syncMode.val();
+				let $syncMode     	   = $( selectElement );
+				let syncModeValue 	   = $syncMode.val();
+				let isProductPublished = !! facebook_for_woocommerce_products_admin.is_product_published;
```

Comment:
Thanks, @message-dimke,

I used it because is_product_published is passed as an empty string ("") for false and the string "1" for true. The double negation converts the string to a true boolean value.

Reply:
No need to change this, I just noted that internally those are two equal operations, nevermind.